crates/tuono/src/watch.rs

```diff
@@ -114,6 +122,15 @@ pub async fn watch() -> Result<()> {
                 if file_path.ends_with("sx") || file_path.ends_with("mdx") {
                     should_reload_ssr_bundle = true
                 }
+
+                if path
+                    .0
+                    .file_name()
+                    .map(|f| f.to_string_lossy().starts_with(".env"))
+                    .unwrap_or(false)
+                {
+                    should_reload_ssr_bundle = true
+                }
```

Comment:
Better export this condition to a function and merge it with the other `should_reload_rust_server` logic. This for keeping a single source of thruth